src/js/clickable-component.js

```diff
@@ -121,14 +125,14 @@ class ClickableComponent extends Component {
    * @return {Component} The child component (created by this process if a string was used)
    * @method addChild
    */
-  addChild(child, options={}) {
-    // TODO: Fix adding an actionable child to a ClickableComponent; currently
+  addChild(child, options = {}) {
+    // TO_DO: Fix adding an actionable child to a ClickableComponent; currently
```

Comment:
I think the TODO rule is just a warning, should we just keep the warnings printing and work on fixing whatever it is we need to do?